index.tmpl.html

```diff
@@ -25,8 +25,8 @@ <h2 id="haskell-on-macos">haskell-on-macos.py</h2>
 
 <pre>
 brew upgrade python@3
-./haskell-on-macos.py --make-dirs --paths.d --ghc-alias=8.8.3 --cabal-alias=3.0.0.0 install ghc-8.0.2 ghc-8.2.2 ghc-8.4.4 ghc-8.6.5 ghc-8.8.3 cabal-install-3.0.0.0
-cabal new-build -w ghc-8.8.3 all
+./haskell-on-macos.py --make-dirs --paths.d --ghc-alias=8.10.1 --cabal-alias=3.0.0.0 install ghc-8.4.4 ghc-8.6.5 ghc-8.8.3 ghc-8.10.1 cabal-install-3.0.0.0
```

Comment:
I wouldn't suggest using `ghc-8.10.1` as a default (i.e. `--ghc-alias=8.10.1`) just yet.

Reply:
I was thinking that people would always want the latest and the greatest as default, but I guess you are right that it can wait for .2 at least 😄

Reply:
And the dependencies to be there...